ui/frontend/AdvancedEditor.tsx

```diff
@@ -11,7 +11,7 @@ const displayExternCrateAutocomplete = (editor: any, autocompleteOnUse: boolean)
   const precedingText = line.slice(0, pos.column);
 
   return !!precedingText.match(/extern\s+crate\s*\w*$/) ||
-    (autocompleteOnUse && !!precedingText.match(/use\s+(?!crate|self|super)\w*$/));
+    (autocompleteOnUse && !!precedingText.match(/^use\s+(?!crate|self|super)\w*$/));
```

Comment:
Would you mind making the same change for `extern crate` (on the line above)?

Reply:
And actually, I should have specified that I actually meant "start of line (with optional whitespace)". For example, this should still bring it up:

```rust
fn foo() {
    use 
}
```